website_sale_one_step_checkout/models/res_config.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class WebsiteConfigSettings(models.TransientModel):
+    """Settings for the OSC."""
```

Comment:
I just checked the OCA repos and I think we can remove the class description (it is not use in any add-on). So please remove all class comments.